browser/ui/webui/private_new_tab_page/brave_private_new_tab_page_handler.cc

```diff
@@ -93,6 +94,17 @@ void BravePrivateNewTabPageHandler::GetIsTorConnected(
   std::move(callback).Run(is_connected);
 }
 
+void BravePrivateNewTabPageHandler::GetIsTorDisabled(
+    GetIsTorDisabledCallback callback) {
+  DCHECK(profile_);
```

Comment:
nit: I believe CHECK() is preferred over DCHECK() 

https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/checks.md#invariant_verification-mechanisms

Reply:
I'll follow up with a separate PR for that just to keep the concerns separated that renames them all in this area